openlcsd/flow/tasks.py

```diff
@@ -838,7 +838,7 @@ def save_package_data(context, engine):
             'task_id': context.get('task_id')
         }
         json.dump(file_content, destination, cls=DateEncoder)
-    resp = cli.post(url, data={"file_path": tmp_file_path}, timeout=60)
+    resp = cli.post(url, data={"file_path": tmp_file_path}, timeout=300)
```

Comment:
We have two pos here, one is posting source data, and the other is post scan result